modules/yieldoneBidAdapter.js

```diff
@@ -0,0 +1,70 @@
+import * as utils from 'src/utils';
+import {config} from 'src/config';
+import {registerBidder} from 'src/adapters/bidderFactory';
+
+const BIDDER_CODE = 'yieldone';
+const ENDPOINT_URL = '//y.one.impact-ad.jp/h_bid';
+
+export const spec = {
+  code: BIDDER_CODE,
+  aliases: ['y1'],
+  isBidRequestValid: function(bid) {
+    return !!(bid.params.placementId);
+  },
+  buildRequests: function(validBidRequests) {
+    return validBidRequests.map(bidRequest => {
+      const params = bidRequest.params;
+      const width = bidRequest.sizes[0];
```

Comment:
this won't work with multi size arrays. Example sizes = [[300,250], [728,90]]

take a look at https://github.com/prebid/Prebid.js/blob/master/src/utils.js#L113 for help with parsing.

Reply:
Hello @mkendall07 

Please tell me what kind of case is multi-size with same Placement ID.

Reply:
you don't have to read multiple sizes if you dont want, but you should take the first size if it's multiple.

Reply:
Hello @mkendall07

I fixed it, so please check it.